docs/developer-guide.md

```diff
@@ -11,31 +11,40 @@ On Windows you can just open the provided solution file `src/OpenSage.sln` to ge
 ### Linux & MacOS
 
 On Unix systems you want to use the .NET Core CLI tools to compile OpenSage:
-```
+
+```shell
 cd src
 dotnet build
 ```
 
 You can also build like that on Windows, in case you don't want to use Visual Studio.
 
 #### Linux dependencies
- - .NET 6.0.300+
- - `libc`
- - `libsdl2`
- - `libopenal`
+
+- .NET 6.0.300+
+- `libc`
+- `libsdl2`
+- `libopenal`
 
 ##### .NET 6.0.300+
-The standard package manage install for dotnet on linux currently installs 6.0.1xx, which does not contain some of the C# 11 preview features used in OpenSAGE. To install 6.0.402 or greater on Ubuntu, follow the instructions [here](https://github.com/dotnet/core/issues/7699) for using a .NET 6 package via **PMC** (_not_ via Jammy feed).
+
+The standard package manage install for dotnet on linux currently installs 6.0.1xx, which does not contain some of the C# 11 preview features used in OpenSAGE. To install 6.0.402 or greater on Ubuntu, follow the [instructions here](https://github.com/dotnet/core/issues/7699) for using a .NET 6 package via **PMC** (_not_ via Jammy feed).
+
+In RPM based systems you may follow the instructions on the official [installation page](https://learn.microsoft.com/en-us/dotnet/core/install/linux-fedora?tabs=dotnet9).
```

Comment:
Seems like this links to .NET 9, which isn't the version used by this project. This project uses .NET 8 (.NET 9 is not an LTS version; only the even-numbered versions are LTS).

Of course, there's the separate issue of all this documentation saying .NET 6, which should not be used and is EOL as of November 2024...

Reply:
If you change the url to `https://learn.microsoft.com/en-us/dotnet/core/install/linux-fedora?tabs=dotnet8` then the install instructions default to .NET 8 instead of 9